File: chart/src/main/java/cn/jingzhuan/lib/chart2/event/OnViewportChangeListener.java
package cn.jingzhuan.lib.chart2.event;

import cn.jingzhuan.lib.chart2.Viewport;

/**
 * Created by Donglua on 17/7/24.
 */
@FunctionalInterface
public interface OnViewportChangeListener {

    void onViewportChange(Viewport viewport);

}

File: chart/src/main/java/cn/jingzhuan/lib/chart3/event/OnViewportChangeListener.kt
package cn.jingzhuan.lib.chart3.event

import cn.jingzhuan.lib.chart3.Viewport

/**
 * @since 2023-09-08
 * @author lei
 * viewport变化监听
 */
fun interface OnViewportChangeListener {

    fun onViewportChange(viewport: Viewport)

}